src/build.props

```diff
@@ -19,7 +19,7 @@
     <Company Condition=" '$(Company)' == '' ">Microsoft</Company>
     <Product Condition=" '$(Product)' == '' ">Microsoft SARIF Viewer for Visual Studio</Product>
     <Copyright Condition=" '$(Copyright)' == '' ">© Microsoft Corporation. All rights reserved.</Copyright>
-    <VersionPrefix Condition=" '$(VersionPrefix)' == ''">2.1.14</VersionPrefix>
+    <VersionPrefix Condition=" '$(VersionPrefix)' == ''">2.1.15</VersionPrefix>
```

Comment:
>15 [](start = 61, length = 2)

Revert: 2.1.14 not yet published.

Reply:
Nope: some insiders have already installed it.

---
In reply to: [489624314](https://github.com/microsoft/sarif-visualstudio-extension/pull/195#discussion_r489624314) [](ancestors = 489624314)